etl/steps/data/garden/tourism/2023-05-05/unwto.meta.yml

```diff
@@ -14,177 +11,313 @@ dataset:
     date_accessed: '2023-05-04'
     publication_date: '2023-01-25'
     publication_year: 2023
-    published_by: UNWTO, 2023
+    published_by: World Tourism Organization (2023), UNWTO Tourism Statistics Database, Madrid, data updated on 25/01/2023.
 tables:
   unwto:
     variables:
       in_to_ar_ov_vi_to:
         title: Inbound arrivals (tourists)
         unit: ''
+        description: >
+          When an individual visits a country multiple times within a year, each visit is counted as a separate arrival. Similarly, if a person travels to multiple countries during a single trip, their arrival in each country is recorded individually. As a result, the number of arrivals does not necessarily reflect the number of individuals traveling, as one person can generate multiple arrivals.
+
+
+          Tourist arrivals data pertain to international visitors entering the economic territory of the referenced country and include visits by people who stay overnight. These data can be obtained from various sources, including administrative records such as police, immigration, and traffic controls, as well as border surveys and registrations at accommodation establishments.
         display:
           name: International tourist arrivals
           numDecimalPlaces: 0
+
       in_to_ar_sa_da_vi_ex:
         title: Inbound arrivals (same day)
-        description: Note that before 2006, the US counted same-day trips from Canada
-          only. After 2006, day trips from Mexico were also included.
         unit: ''
+        description: >
+          When an individual visits a country multiple times within a year, each visit is counted as a separate arrival. Similarly, if a person travels to multiple countries during a single trip, their arrival in each country is recorded individually. As a result, the number of arrivals does not necessarily reflect the number of individuals traveling, as one person can generate multiple arrivals.
+
+
+          Same-day arrivals data pertain to international visitors entering the economic territory of the referenced country and only staying for the day. These data can be obtained from various sources, including administrative records such as police, immigration, and traffic controls, as well as border surveys and registrations at accommodation establishments.
         display:
           name: International same-day arrivals
           numDecimalPlaces: 0
+
       do_to_ac_ho_an_si_es_gu:
-        title: Domestic guests in tourism-accommodation (hotels and similar establishments)
+        title: Domestic guests in tourism accommodation (hotels and similar establishments)
         unit: ''
+        description: >
+          The indicator refers to the number of domestic guests who stay in tourism accommodation establishments such as hotels and similar accommodations. This indicator focuses specifically on individuals from the same country as the location of the accommodation.
```

Comment:
I suspect people taking several trips are again counted several times? It may be worth mentioning that.